src/App.tsx

```diff
@@ -26,70 +27,127 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+};
+
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((item1, item2) => item1.length - item2.length);
+  }
+
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort();
+  }
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
```

Comment:
limited number of pre-defined conditions. You sure `if` is the best solution here?